### ISSUE TITLE
add method for removing event listener

### ISSUE THREAD

**Issue by Mohmn** [Greenstand/treetracker-web-map-core#134]
---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-core#treetracker-web-map-core) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.

**Comment by dadiorchen** [comment]
Yes, this is good, now we are using node built-in (if I can say this way) event API to handle the event listener, so it seems easy to expose something @Mohmn required in this issue.

https://nodejs.org/api/events.html

**Comment by Mohmn** [comment]
@dadiorchen yep its pretty straight forward, should i do it myself or leave it for the new contributors

**Comment by dadiorchen** [comment]
I hope you can take it, it is not a easy one

**Comment by Mohmn** [comment]
> I hope you can take it, it is not a easy one

I will give it a go then
